apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -255,6 +255,9 @@ export function formatProsecutorCourtDateEmailNotification(
   const courtRoomText = formatMessage(notifications.courtRoom, {
     courtRoom: courtRoom || 'NONE',
   })
+
+  const nonePresentCourtRoom = formatMessage(cf.nonePresentSession)
```

Comment:
It woul probably be better to combine `courtRoomText` and `nonPresentCourtRoom` and just use `courtRoomText`. Otherwise, the Contentful string for `body` needs to be changed to include a variable `nonPresentCourtRoom`.